src/main/resources/db/changelog/changes/02-add-default-value-to-boolean-field-book-table.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-default-value-to-is-deleted-field-in-book-table
```

Comment:
it makes no sense to add additional script for just a default value for a column - you can specify it in the main script, where you create books table